packages/backend/media-optimiser/README.md

```diff
@@ -0,0 +1,20 @@
+<h1 align="center">Gravitywell Lambda Utils</h1>
+<p align="center">Library of Lambda utilities</p>
+<p align="center">
+  <img src="https://img.shields.io/github/workflow/status/GravitywellUK/packages/CI/master" alt="CI" />
+  <img src="https://img.shields.io/github/license/gravitywelluk/packages" alt="License" />
+  <img src="https://img.shields.io/npm/dm/@gravitywelluk/media-optimiser" alt="Downloads" />
+  <img src="https://img.shields.io/npm/v/@gravitywelluk/media-optimiser" alt="Version" />
+</p>
+<br />
+
+## Usage
+
+```typescript
+const mediaOptimiser = require('media-optimiser');
+
+const transformedMedia = mediaOptimiser.optimiseMedia({
+  fileName,
+  fileBuffer
+})
+```
```

Comment:
I think in this file we should inform the user about pre-requisites about the pre-built binaries - https://sharp.pixelplumbing.com/install. This will affect the types of images Sharp is able to process and will cause it to vary from machine to machine